semeio/jobs/scripts/overburden_timeshift.py

```diff
@@ -38,14 +39,13 @@ def _get_args_parser():
     return parser
 
 
-def main_entry_point(args=None):
+def main_entry_point():
```

Comment:
If we want to test the `main_entry_point` without modifying `sys.args` at runtime (and without using `subprocess`), we want to allow the `main_entry_point` to accept arguments I believe.

Reply:
Dont want to adapt production code to adhere to tests, can always mock `sys.args`?

Reply:
```
import argparse
import sys
def test_argparse(monkeypatch):
    monkeypatch.setattr(sys, "argv", ["path", "-c", "config_file"])
    parser = argparse.ArgumentParser()
    parser.add_argument(
        "-c", "--config"
    )
    options = parser.parse_args()
```

Reply:
Alright, that's a reasonable way of doing it, I think. And of course we're only mocking specifically sysargv, and not anything else